### ISSUE TITLE
Non-square maxpooling for TensorFlow-->IR-->Caffe

### ISSUE THREAD

**Issue by eerobert** [Microsoft/MMdnn#494]
Platform (like ubuntu 16.04/win10): ubuntu 16.04

Python version: 2.7

Source framework with version (like Tensorflow 1.4.1 with GPU): 1.12 GPU

Destination framework with version (like CNTK 2.3 with GPU): Caffe

Pre-trained model path (webpath or webdisk path): NA

Running scripts: NA

In tensorflow network I have a layer of 
    max_pool(x, ksize=[1, 2, 1, 1], strides=[1, 2, 1, 1], padding='SAME')
after convert to IR then Caffe, it becomes 
    layer { ... type: "Pooling" pooling_param{kernel_size: 2...}}

Just wonder the current release do support non-square max_pool?

Thanks

**Comment by rainLiuplus** [comment]
Hi @eerobert . Yes, current release does not support it. We are going to fix it and will ping you if it is done.
